aws/resource_aws_rds_cluster.go

```diff
@@ -222,6 +224,29 @@ func resourceAwsRDSCluster() *schema.Resource {
 				Optional: true,
 			},
 
+			"iam_role": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"role_arn": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"role_status": {
+							Type:     schema.TypeString,
+							Computed: true,
```

Comment:
@apparentlymart this is purely for information only - it's nothing to do with the actual diff. It just allows people to see the status of a role.

FYI, the feedback from the original PR was to add this - thus i've added it